setup.cfg

```diff
@@ -30,6 +30,7 @@ install_requires =
     sunpy>=4.0.1
     watchdog
     requests
+    stixdcpy @ git+ssh://git@github.com/i4ds/stixdcpy.git
```

Comment:
Is there a tag or version we can set here? This will just install what ever is on master